modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -63,6 +63,10 @@ public static long addListTypeDefinition(
 			return 0;
 		}
 
+		if (Validator.isNotNull(objectField.getListTypeDefinitionId())) {
+			return objectField.getListTypeDefinitionId();
+		}
```

Comment:
@murilostodolni do we need to check if the `objectField.getListTypeDefinitionId()` is valid?

Reply:
Hi @carolmariaabb, thanks for the question. The validation if listTypeDefinition exist with this ID is made in service.